src/test/java/calculator/OperatorTest.java

```diff
@@ -50,4 +51,16 @@ void divideTest() {
         assertThat(2).isEqualTo(Operator.findSymbol(formula.get(1))
                 .calculate(Integer.parseInt(formula.get(0)), Integer.parseInt(formula.get(2))));
     }
+
+    @DisplayName("모든 숫자는 0으로 나눌 수 없다.")
+    @Test
+    void divideZeroTest() {
+        formula.add("6");
+        formula.add("/");
+        formula.add("0");
+        assertThatIllegalArgumentException().isThrownBy(() -> {
+            assertThat(0).isEqualTo(Operator.findSymbol(formula.get(1))
+                    .calculate(Integer.parseInt(formula.get(0)), Integer.parseInt(formula.get(2))));
+        });
```

Comment:
`formula`라는 `List` 타입의 변수에 값을 넣은뒤, 그것들을 꺼내서 테스트를 진행하는 대신, 아래와 같이 테스트를 진행해볼 수 있을 것 같아요. 그러면 `formula`를 선언하지 않아도 되겠죠~? :)
```suggestion
       assertThatIllegalArgumentException()
                .isThrownBy(() -> Operator.findSymbol("/").calculate(6, 0));
```
추가로, `assertThat(actual).isEqualTo(expected)`와 같이 사용하도록 해요. `actual`에는 `테스트하고자하는 값`, `expected`에는 `테스트의 결과로 기대하는 값`을 넣으면 된답니다.
이 부분은, 실패하는 테스트를 작성한 뒤, 콘솔의 출력이 어떻게 달라지는지 확인해보면 좋을 것 같아요! :)